### ISSUE TITLE
Freundlicheres Banner finden

### ISSUE THREAD

**Issue by matthias-mueller** [pikobytes/smartrain-website#7]
Die Clkipart-Grafik mit den fetten grauen Regenwolken passt zwar inhaltlich ganz gut, ist aber ziemlich dunkel und tiefgrau. Vielleiucht finden wir ja hier nochwas schöneres, z.B. Regen und Sonne oder eine aufziehende [Cumuluswolke](http://deacademic.com/pictures/dewiki/65/Anvil_shaped_cumulus_panorama_edit_crop.jpg)

**Comment by PIEKAR** [comment]
Das "große" Bild würde ich heute Abend zu Hause mal etwas kleiner machen. Ich finde aber auch, dass ein passenderes Bild schöner wäre. Matthias, das was du geschickt hast passt schon eher. Wenn es aber etwas dresdenspezifischer wäre, wäre das glaube ich etwas besser. Ich kann ja mal Daniel fragen ob er ne Idee hat.

**Comment by jacmendt** [comment]
Hat sich bei diesem Thema schon ein Fortschritt ergeben?

**Comment by PIEKAR** [comment]
HAb ich erledigt

**Comment by jacmendt** [comment]
Herzlichen Dank. Dann können wir das Ticket ja schließen.

**Comment by matthias-mueller** [comment]
@jacmendt Nein. Das Hauptproblem ist nicht die Bildgröße, sondern das Motiv.

**Comment by PIEKAR** [comment]
Ja ich weiß... mir ist aber noch nichts besseres begegnet. Eigene Regenbilder könnte ich von Island anbieten. Dresdner BIlder gab es in diesem Jahr ja keine.
